scripts/screenshots.js

```diff
@@ -29,6 +29,7 @@ const defaultGetScreenshotName = ({ component, viewport }) =>
 
 const formatViewport = viewport => `${viewport.width}x${viewport.height}`
 
+const pushAll = (arr, items) => arr.push.apply(arr, items)
```

Comment:
Nit, but here's a little trick: `array.push` accepts an infinite number of args, so `arr.push(...items)` does the same :)

Reply:
Yes but it also makes a copy of the array and also needs a tad bit more code ;)

<img width="1144" alt="image" src="https://user-images.githubusercontent.com/465582/78535009-b53e4b80-77eb-11ea-8234-29d541b5225b.png">